zcash_client_backend/src/keys.rs

```diff
@@ -136,7 +135,7 @@ impl Era {
     }
 }
 
-/// A set of spending keys that are all associated with a single
+/// A set of viewing keys that are all associated with a single
```

Comment:
This says viewing keys, but the struct is still named `UnifiedSpendingKey` and refers to other spending key types, which seems wrong.